### ISSUE TITLE
Get area of polygon

### ISSUE THREAD

**Issue by pierluigiderosa** [geopaparazzi/smash#172]
I need to get the area of a digitized polygon in SMASH.
Is there a way to get such information? I remember in geopaparazzi this was possible.
Thaks for the anwer
P

**Comment by moovida** [comment]
Hi @pierluigiderosa , I was sure I had added it in the attributes table, but that was probably geopaparazzi. I sure want to add area and perimeter to polygons and lines in the info box with the attrbutes.

Will keep this issue open to keep track of it. Thanks

**Comment by pierluigiderosa** [comment]
Thanks @moovida. I wait for the update.
Thanks for the feature

**Comment by pierluigiderosa** [comment]
Thanks @moovida
I Will find this feature in the next release?

Il Lun 14 Mar 2022, 14:49 Andrea Antonello ***@***.***> ha
scritto:

**Comment by moovida** [comment]
Yes, exactly. And I am trying real hard to get one out very very soon.
